azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/features/ACSFeature.kt

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.features
+
+/**
+ * Expose common actions between the ACS features
+ *
+ * @author yassirb@microsoft.com
+ */
+internal interface ACSFeature {
+    /**
+     * Return feature availability
+     *
+     * @return true if available on the current release otherwise false
+     */
+    fun IsAvailable(): Boolean
```

Comment:
should it be `isAvailable` get only property?

Reply:
will update, I pulled this code direct from calling native